### DISCUSSION TITLE
买了Daisyblocks，图库1.3.0在Halo2.15.2上不显示

### DISCUSSION THREAD

**Discussion by lomiles** [halo-dev/halo#5904] [General]
![微信截图_20240513160411](https://github.com/halo-dev/halo/assets/35911812/289d2ccb-01a7-4f9a-8c29-cc7cdb3776d5)
![微信截图_20240513160548](https://github.com/halo-dev/halo/assets/35911812/d38331c7-862a-458b-8bec-2720e07f9124)

**Comment by wan92hen** [comment]
![image](https://github.com/halo-dev/halo/assets/27671436/b5177114-5db6-4367-9158-0d3ee2223a22)

**Reply by lomiles** [reply]
可以了，thx
